src/visibility-and-privacy.md

```diff
@@ -134,8 +141,7 @@ pub mod submodule {
         }
     }
 }
-
-# fn main() {}
+fn main(){};
```

Comment:
These changes should be undone, it breaks the tests.

Reply:
i only added that because the previous version felt like the hashtag was made as a comment and the next example had it.

Reply:
ok, changed and now it passes the tests.